src/pudl/transform/ferc1.py

```diff
@@ -774,7 +773,7 @@ def _plants_steam_clean(ferc1_steam_df):
         .pipe(pudl.helpers.cleanstrings, ['construction_type', 'plant_type'], [CONSTRUCTION_TYPE_STRINGS, PLANT_KIND_STRINGS], unmapped=pd.NA)
         .pipe(pudl.helpers.oob_to_nan,
               cols=["construction_year", "installation_year"],
-              lb=1850, ub=max(pc.WORKING_PARTITIONS["ferc1"]["years"]) + 1)
+              lb=1850, ub=max(Ferc1Settings.working_partitions["years"]) + 1)
```

Comment:
Indirect references to copied values rather than the canonical values in the DataSource.